rfd/0087/README.md

```diff
@@ -33,9 +33,11 @@ run` with Docker 1.13.0.
 
 ## What is /events and why does it matter?
 
-At the time of this writing Docker's documentation for events is
-[here](https://docs.docker.com/engine/api/v1.25/#operation/SystemEvents) though
-recent changes seem to indicate that this URL is not permanent.
+At the time of this writing, it's not possible to link to Docker's documentation
```

Comment:
OK, so it seems the problem I had with not seeing the proper section of the documentation when clicking on that link was specific to Opera and Chrome. It seems to be working with a browser based on Firefox's engine (Cliqz in my case).